source/exercises/string_formatting.rst

```diff
@@ -130,15 +136,63 @@ Task Three
 
     use string formating to print:
 
-    ``'02 27 2017 04 30``
+    ``'02 27 2017 04 30'``
 
 Hint: use index numbers to specify positions.
 
 
-Task four
+Task Five
 ---------
+f-strings are new to Python (version 3.6), but are very powerful and efficient. This means they are worth understanding and using. And this is made easier than it might be because they use the same, familiar formatting language that is conventionally used in Python (in .format()).
+
+So in this exercise we are going to specifically use f-strings.
+
+Here's the simplest example, just to show the basic syntax:
```

Comment:
I'm not sure we should have an example if an f-string with no formatting -- kinda makes you wonder what the difference is -- and the answer is "nothing" :-)

So let's leave this out and go straight to the example with {name}